independent-publisher-connectors/Mensagia/Readme.md

```diff
@@ -0,0 +1,37 @@
+# Mensagia
+Mensagia offer reliable SMS, Email and Voice for business automation communication.
+
+## Publisher: Sinergyne Global Communications, S.L.
+
+## Prerequisites
+You will need the following to proceed:
+* A Mensagia "API Client" account.
+* Make sure you have enough balance in your Mensagia account.
+
+## Obtaining Credentials
+The API key can be obtained by entering the portal of your Mensagia account.
```

Comment:
@AfriSR You need to provide step-by-step details to get an API that can be understood by everyone.